client/js/assets/fonts/stylesheet.css

```diff
@@ -0,0 +1,176 @@
+        url('NeueHaasDisplay-Black.ttf') format('truetype');
+    font-weight: 900;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'Neue Haas Grotesk Display Pro';
+    src: url('NeueHaasDisplay-XXThin.eot');
+    src: local('Neue Haas Grotesk Display Pro 15 XXThin'), local('NeueHaasDisplay-XXThin'),
+        url('NeueHaasDisplay-XXThin.eot?#iefix') format('embedded-opentype'),
+        url('NeueHaasDisplay-XXThin.woff') format('woff'),
+        url('NeueHaasDisplay-XXThin.ttf') format('truetype');
+    font-weight: 100;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'Neue Haas Grotesk Display Pro';
+    src: url('NeueHaasDisplay-MediumItalic.eot');
+    src: local('Neue Haas Grotesk Display Pro 66 Medium Italic'), local('NeueHaasDisplay-MediumItalic'),
+        url('NeueHaasDisplay-MediumItalic.eot?#iefix') format('embedded-opentype'),
+        url('NeueHaasDisplay-MediumItalic.woff') format('woff'),
+        url('NeueHaasDisplay-MediumItalic.ttf') format('truetype');
+    font-weight: 500;
+    font-style: italic;
+}
+
+@font-face {
+    font-family: 'Neue Haas Grotesk Display Pro';
+    src: url('NeueHaasDisplay-Bold.eot');
+    src: local('Neue Haas Grotesk Display Pro 75 Bold'), local('NeueHaasDisplay-Bold'),
+        url('NeueHaasDisplay-Bold.eot?#iefix') format('embedded-opentype'),
+        url('NeueHaasDisplay-Bold.woff') format('woff'),
+        url('NeueHaasDisplay-Bold.ttf') format('truetype');
+    font-weight: bold;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'Neue Haas Grotesk Display Pro';
+    src: url('NeueHaasDisplay-XThin.eot');
+    src: local('Neue Haas Grotesk Display Pro 25 XThin'), local('NeueHaasDisplay-XThin'),
+        url('NeueHaasDisplay-XThin.eot?#iefix') format('embedded-opentype'),
+        url('NeueHaasDisplay-XThin.woff') format('woff'),
+        url('NeueHaasDisplay-XThin.ttf') format('truetype');
+    font-weight: 100;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'Neue Haas Grotesk Display Pro';
+    src: url('NeueHaasDisplay-LightItalic.eot');
+    src: local('Neue Haas Grotesk Display Pro 46 Light Italic'), local('NeueHaasDisplay-LightItalic'),
+        url('NeueHaasDisplay-LightItalic.eot?#iefix') format('embedded-opentype'),
+        url('NeueHaasDisplay-LightItalic.woff') format('woff'),
+        url('NeueHaasDisplay-LightItalic.ttf') format('truetype');
+    font-weight: 300;
+    font-style: italic;
+}
+
+@font-face {
+    font-family: 'Neue Haas Grotesk Display Pro';
+    src: url('NeueHaasDisplay-XThinItalic.eot');
+    src: local('Neue Haas Grotesk Display Pro 26 XThin Italic'), local('NeueHaasDisplay-XThinItalic'),
+        url('NeueHaasDisplay-XThinItalic.eot?#iefix') format('embedded-opentype'),
+        url('NeueHaasDisplay-XThinItalic.woff') format('woff'),
+        url('NeueHaasDisplay-XThinItalic.ttf') format('truetype');
+    font-weight: 100;
+    font-style: italic;
+}
+
```

Comment:
What does this file do? If you are adding just fonts, you do not need this file.

Reply:
Made the following changes. Thank you.